workflow-templates/update_rubocop.yml

```diff
@@ -0,0 +1,57 @@
+name: Update Rubocop
+
+# This action automatically updates the rubocop gems within a
+# repository and attempt to ensure that the resulting build passes.
+# In order to use it effectively on a given project you should ensure:
+# * All Rubocop associated gems are in the 'lint' group in the Gemfile
+# * You update the 'BUNDLE_WITHOUT' section to exclude other groups, especially
+#   those which will block `bundle install`
+#
+# Once added this will add an actions which:
+# * Automatically runs at 10 am Sunday
+# * Can be triggered manually via a workflow_dispatch (See link below)
+# * Will automatically update all gems in the 'lint' group
+# * Will run new safe autocorrect cops, and will add inline rubocop:todo for
+#   other cops
+# * Will automatically generate a pull request with the above changes.
+#
+on:
+  # Can be triggered manually from the GH interface or via a dispatches POST
+  # https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
+  workflow_dispatch:
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+  # Every sunday at 10 am
+  - cron: "0 10 * * 0"
+
+jobs:
+  update_rubocop:
+    if: (github.repository_owner == 'sanger') # On the sanger fork only
+    runs-on: ubuntu-latest
```

Comment:
Should we pin this to what we use in FCE instead or is it low risk?

Reply:
For linting alone, I'm going to say pretty low risk.

I haven't been version pinning the test suites, perhaps we should. When I started the process of migrating, they didn't even offer version pinning, or it was poorly documented, that got added as they started the migration from 18.0.4 to 20.04)

That said, it should still be fairly low risk for most the SS stuff, as we're not actually packaging the binaries we generate as part of the build.